scripts/enum/status.lua

```diff
@@ -3,7 +3,7 @@ xi = xi or {}
 xi.status =
 {
     NORMAL          =  0,
-    MOB             =  1,
+    UPDATE          =  1,
```

Comment:
we have a few other locations where xi.status.MOB is being referenced. We'll need to update there too. Looks like:
- se_apollyon.lua
- sw_apollyon.lua
- voidwalker.lua
- qiqirn_treasure_hunter.lua
- etc. etc.

Reply:
nvm I see this is handled in another PR